tests/e2e/kafka.go

```diff
@@ -48,3 +48,26 @@ func requireDeployingKafkaTopic(kubectlOptions k8s.KubectlOptions, topicName str
 	})
 
 }
+
+// requireCreatingKafkaUser creates a KafkaUser resource from a template
+func requireCreatingKafkaUser(kubectlOptions k8s.KubectlOptions, userName string, tlsSecretName string) {
+	It("Deploying KafkaUser CR", func() {
+		templateParameters := map[string]interface{}{
+			"Name":      userName,
+			"Namespace": kubectlOptions.Namespace,
+		}
+		if tlsSecretName != "" {
+			templateParameters["TLSSecretName"] = tlsSecretName
+		}
```

Comment:
**Note**: In the future we might want to extract these into a `kafkaUser(userName, namespace, optionalTLSSecretName string) (KafkaUser, error)` function, looks reusable.

Reply:
Hm, fair, I'll add it to the internal refactor ticket if that is okay.

Reply:
Sure